src/app/components/carousel/carousel.ts

```diff
@@ -613,14 +609,23 @@ export class Carousel implements AfterContentInit {
                 }
             }
         }, this.autoplayInterval);
+        this.allowAutoplay = true;
     }
 
-    stopAutoplay() {
+    stopAutoplay(changeAllow: boolean = true) {
         if (this.interval) {
             clearInterval(this.interval);
+            this.interval = undefined;
+            if(changeAllow){
+                this.allowAutoplay = false;
+            }
         }
     }
 
+    isPlaying(){
+        return this.interval;
```

Comment:
Hi, can you convert to `boolean`, because now it is a number or undefined, for understanding is not clear.

Reply:
Done 👍